R/plot_trace.R

```diff
@@ -3,11 +3,40 @@
 #' @param data An object of class [mice::mids].
 #' @param vrb String, vector, or unquoted expression with variable name(s), default is "all".
 #'
+#' @details
+#' The `vrb` argument is "quoted" via [rlang::enexpr()] and evaluated according
+#' to [Tidy Evaluation principles](https://adv-r.hadley.nz/metaprogramming.html).
+#' In practice, this technical nuance only affects users when passing an object
+#' from the environment (e.g., a vector of variable names) to the `vrb` argument.
+#' In such cases, the object must be "unquoted" via the `!!` prefix operator.
+#'
 #' @return An object of class [ggplot2::ggplot].
 #'
 #' @examples
 #' imp <- mice::mice(mice::nhanes, print = FALSE)
+#'
+#' ## Plot all imputed variables
 #' plot_trace(imp)
+#'
+#' ## Variables can be specified via character vectors comprising their names
+#' plot_trace(imp, "bmi")
+#' plot_trace(imp, c("bmi", "hyp"))
+#'
+#' ## Variable names can be unquoted
+#' plot_trace(imp, bmi)
+#' plot_trace(imp, c(bmi, hyp))
+#'
+#' ## When passing the variable names as an object from the environment, the
+#' ## object's name must be unqoted via `!!`.
+#' vars <- c("bmi", "hyp")
+#' plot_trace(imp, vars) |> try() # Error
+#' plot_trace(imp, !!vars)        # Runs because the 'vrb' argument is unquoted
+#'
+#' for(v in vars)
+#'   plot_trace(imp, !!v) |> print()
+#'
+#' @importFrom utils tail
```

Comment:
`ggmice` adheres to the https://r-pkgs.org/ recommendation of using `::` for imported functions instead of `importFrom`